low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -273,13 +260,7 @@ class SSPLDevDeploy:
 
     def setup_kafka(self):
         """Setup kafka"""
-        build_url = f"{self.build_url}/3rd_party/commons/kafka/"
-
-        pkg = [re.findall("<a .*>(.*?)</a>", str(line))[0] for line in 
-               urllib.request.urlopen(build_url).readlines()
-               if re.search(r"kafka.*\.rpm", str(line))][0]
-
-        cmd = f"yum install -y {build_url}{pkg}"
+        cmd = f"yum install -y --nogpgcheck kafka"
```

Comment:
why nogpg check again for 3rd party kafka ? And why kafka is being deployed independently when cortx-prereqs covers it ?

Reply:
cortx prereqs only install python third party packages. Kafka is not part of it. Also Kafka RPM is build locally from binaries, so we need nogpgcheck.

Reply:
`f""` is not required.

Reply:
ok

Reply:
Done.